packages/@webex/plugin-meetings/src/roap/index.ts

```diff
@@ -170,7 +170,7 @@ export default class Roap extends StatelessWebexPlugin {
           locusSelfUrl: meeting.selfUrl,
           mediaId: options.mediaId,
           correlationId: options.correlationId,
-          audioMuted: meeting.isAudioMuted(),
+          audioMuted: meeting.audio && meeting.audio.isLocallyMuted(),
```

Comment:
minor thing, but doing `meeting.audio?.isLocallyMuted()` would be shorter and work just as well right?

Reply:
Changed everywhere Marcin. Thanks :-)